packages/synapse-react-client/src/components/WikiMarkdownEditorButton/WikiMarkdownEditorButton.tsx

```diff
@@ -124,6 +128,20 @@ export const WikiMarkdownEditorButton: React.FunctionComponent<
 
   return (
     <>
+      {displayWikiMarkdown && (
+        <>
+          {wikiPage && wikiPage.markdown !== '' ? (
+            <MarkdownSynapse
+              key={wikiPage.markdown}
```

Comment:
Specify a key so that `MarkdownSynapse` shows the latest markdown after the user edits the wiki markdown content